### PR TITLE
Fixes #1841 use named tuple

### LINKED ISSUE #1841: Importer - remove CSVSeparators class and use named tuples

### PR DESCRIPTION
**Pull request by rwmcintosh**
Missed this on the last PR

### CODE REVIEW

File: src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-      public char DecimalSeparator { get; set; }
-   }
-
    public interface ICsvSeparatorSelector
    {
       /// <summary>
       ///    Returns the separator to be used for the reading of the .csv file
       /// </summary>
-      CSVSeparators GetCsvSeparator(string fileName);
+      (char? ColumnSeparator, char DecimalSeparator) GetCsvSeparator(string fileName);
```
Comment (msevestre) [inline src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs:8]: Naming is off I would say. When you retrieve then, they should be like local variables no?
So columnSeparator and decimalSeparator

File: src/OSPSuite.R/Services/DataImporterTask.cs [hunk 2]
```diff
@@ -105,7 +105,7 @@ public bool IgnoreSheetNamesAtImport
          string dataPath,
          char columnSeparator, char decimalSeparator = '.')
       {
-         _csvSeparatorSelector.CsvSeparators = new CSVSeparators { ColumnSeparator = columnSeparator, DecimalSeparator = decimalSeparator };
+         _csvSeparatorSelector.CsvSeparators = (';', ',');
```
Comment (msevestre) [inline src/OSPSuite.R/Services/DataImporterTask.cs:108]: What aren't you using the variables passed as parameters?

File: src/OSPSuite.UI/Services/CsvSeparatorSelector.cs [hunk 3]
```diff
@@ -13,13 +13,13 @@ public CsvSeparatorSelector(IApplicationController applicationController)
...
 
-      public CSVSeparators GetCsvSeparator(string fileName)
+      public (char? ColumnSeparator, char DecimalSeparator) GetCsvSeparator(string fileName)
       {
          using (var csvSeparatorPresenter = _applicationController.Start<ICSVSeparatorSelectorPresenter>())
          {
             csvSeparatorPresenter.SetFileName(fileName);
 
-            return csvSeparatorPresenter.Canceled() ? null : csvSeparatorPresenter.GetCSVSeparators();
+            return csvSeparatorPresenter.Canceled() ? (null, '.') : csvSeparatorPresenter.GetCSVSeparators();
```
Comment (msevestre) [inline src/OSPSuite.UI/Services/CsvSeparatorSelector.cs:22]: This should come from the presenter returning . Shouldn't have to ask cancel or not. Tell don't ask

**Review by msevestre — COMMENTED** [review]
I thi I there is the one place where you are not using the variables to create the tuple. 
I thought from the code changes earlier that there was only one method with this signature. It's already spreading everywheee so I am not convinced that tuple is actually the way to go by the way so I would just forget about this change all together lol and stick with the class instead. Thoughts?

**Comment by rwmcintosh** [comment]
It's fine with me. I'll just close and delete the branch.

**Pull request closed without merging**